examples/generic/generic_systems.rs

```diff
@@ -0,0 +1,11 @@
+use bevy::ecs::component::Component;
```

Comment:
```suggestion
use bevy::prelude::*;
```
Generally using the prelude import is a good starting point and then you should explicitly import things along the way as you need them.